commands/generate-key.js

```diff
@@ -36,7 +40,7 @@ module.exports = {
 
         const { deps: { keyStore } } = near.config;
         const existingKey = await keyStore.getKey(argv.networkId, argv.accountId);
-        if (existingKey) {
+        if (existingKey && !argv.seedPhrase) {
```

Comment:
It will fix just 1 out of ~5 problems we have with generation keys. This is a critical please of code, so I would prefer to fix them all at once.